packages/app/src/apis.ts

```diff
@@ -25,16 +25,6 @@ import {
   GraphQLEndpoints,
 } from '@backstage/plugin-graphiql';
 
-import {
-  TravisCIApi,
-  travisCIApiRef,
-} from '@roadiehq/backstage-plugin-travis-ci';
-
-import {
```

Comment:
Does the `@roadiehq/backstage-plugin-github-pull-requests` plugin also need updating too, or does it already have this registered in the plugin?

Reply:
It already has it registered in the plugin.